src/components/SettingsPopout/SettingsPopout.tsx

```diff
@@ -1,17 +1,30 @@
 import { PropsWithChildren } from 'react';
-import { ActionIcon, Menu } from '@mantine/core';
+import { ActionIcon, MantineStyleProps, Menu } from '@mantine/core';
 
 import { SettingsIcon } from '@/icons/icons';
 
-export function SettingsPopout({ children }: PropsWithChildren) {
+interface Props {
+  popoverWidth?: number;
+  title?: string;
+}
+
+export function SettingsPopout({
+  popoverWidth,
+  title,
+  children,
+  ...styleProps
+}: Props & PropsWithChildren & MantineStyleProps) {
```

Comment:
I'm not necessarily against this way of doing this, but please let's stick to the way we've done it in all other components
```suggestion
interface Props extends MantineStyleProps, PropsWithChildren {
  popoverWidth?: number;
  title?: string;
}

export function SettingsPopout({ popoverWidth, title, children, ...styleProps }: Props) {
```